src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -397,10 +406,15 @@ function getBlockchainsOptions(): Record<
       value: BlockchainEnum.BITCOIN,
     },
   }
+  if (userNetwork !== "near:mainnet") {
+    delete (options as Partial<typeof options>)[BlockchainEnum.NEAR]
```

Comment:
??? delete

Reply:
We need this to remove redundant option for Non-Near users.